### DISCUSSION TITLE
Is it possible to downgrade the Android and iOS versions defined by the capacitor?

### DISCUSSION THREAD

**Discussion by caarloshenrique** [ionic-team/capacitor#4293] [Q&A]
Currently the documentation states that the Capacitor 2.0 supports Android 5.0 or higher and iOS 11+. Is it possible to downgrade to use the project in previous versions?

**Comment by jcesarmobile** [comment]
On Android you can change the `minSdkVersion` from 21 to a lower version, but even if you can install the app on older devices, it won't work because it uses features that weren't available before Android 5.

Similar thing for iOS, while you can technically downgrade the deployment target to install in older iOS versions, the apps won't work because Capacitor uses some features that weren't available before.

And on both cases there are a lot of possibilities that the codes doesn't even compile when you downgrade the minimum versions.
